crm_deduplicate_filter/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Exclude some records from the deduplication
+===========================================
+
+This module extends the possibilities of the contact deduplication allowing
+to filter the applicable set according to several criteria.
+
+For now, only the filter for restricting the deduplication to only companies or
+only contacts is implemented.
```

Comment:
That's not a known issue, but only the scope. The known issue / roadmap should be "Implement more filters for deduplication", but I decided to let it this way.